### ISSUE TITLE
Presentation of: GO term enrichment tools to the cloud from AGR.

### ISSUE THREAD

**Issue by owhite** [dcppc/data-stewards#13]
This thread to serve as a request for a presentation at the next Data Stewards / DCPPC call on Alliance for Genome Resources (AGR) Supplement Aims. Specific Aim 1: "Alliance for Genome Resources (AGR) Supplement Aims"

**Comment by cmungall** [comment]
Will also assign @thomaspd when he accepts invite to join project.

I did a 5 minute presentation at the DC kickoff (I discovered we were doing this the night before, so slides are not very polished). Could be used as a basis:
https://docs.google.com/presentation/d/1UBMhk9ivSbyLsfe_4XwKbtLwVQU5rNXJWxHYOmWoVvI/edit#slide=id.p

**Comment by thomaspd** [comment]
Yes, @cmungall and/or I can do this presentation. What are some possible dates?

**Comment by maryshimoyama** [comment]
I added on another thread that I could do the disease annotator
